src/javascript/app/base/socket_general.js

```diff
@@ -35,9 +36,13 @@ const BinarySocketGeneral = (() => {
             case 'website_status':
                 if (response.website_status) {
                     is_available = /^up$/i.test(response.website_status.site_status);
-                    if (is_available && !BinarySocket.availability()) {
-                        window.location.reload();
-                    } else if (!is_available) {
+                    if (is_available) {
+                        if (!BinarySocket.availability()) {
+                            window.location.reload();
+                        } else if (response.website_status.message) {
+                            Footer.displayNotification(response.website_status.message);
+                        }
+                    } else {
                         Header.displayNotification(response.website_status.message, true);
```

Comment:
I think this line can go away now that `website_status.message` will always go to the Footer.